### PR TITLE
Remove deprecated default subcategories

### PR DESCRIPTION
**Pull request by anotherminh**
## Related Issues

- Closes https://transcend.height.app/T-15814

## Security Implications

_[none]_

## System Availability

- We removed some values from `DefaultDataSubCategoryType` so care should be taken to not reference those values anymore when upgrading the privacy-types major version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "author": "Transcend Inc.",
   "name": "@transcend-io/privacy-types",
   "description": "Core enums and types that can be useful when interacting with Transcend's public APIs.",
-  "version": "3.5.1",
+  "version": "4.0.0",
```
Comment (MadDataScience) [inline package.json:5]: Wondering why this merits a major version bump (as opposed to minor).

**Review by MadDataScience — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (anotherminh) [reply package.json:5]: @MadDataScience it's not really backwards compatible? If downstream clients were referring to this enum it would error out hard?

Comment (MadDataScience) [reply package.json:5]: Ah, makes sense.

**Review by michaelfarrell76 — APPROVED** [review]
nice

**Pull request merged**